src/main/java/com/minecolonies/coremod/colony/buildings/modules/BuildingModules.java

```diff
@@ -459,6 +459,7 @@ public class BuildingModules
       new BuildingEntry.ModuleProducer<>("miner_settings", () -> new SettingsModule()
         .with(AbstractCraftingBuildingModule.RECIPE_MODE, new CrafterRecipeSetting())
         .with(BuildingMiner.FILL_BLOCK, new BlockSetting((BlockItem) Items.COBBLESTONE))
+        .with(BuildingBuilder.BUILDING_MODE, new BuilderModeSetting())
```

Comment:
It was previously intended that the miner did not have this setting, so that they'd always use "default" for predictable builds.  Not that I'd personally object to allowing it to be selectable.

Reply:
Huh, odd the miner was throwing exception that they "want it"

Reply:
Possibly a side effect of the refactor?  They definitely didn't have that setting before.

Reply:
Before, it was indeed intended that they used the default one